src/components/widgets-settings/organizations/organizations.directive.coffee

```diff
@@ -36,21 +36,18 @@ module.controller('SettingOrganizationsCtrl', ($scope, $log, ImpacDashboardsSvc,
         $scope.dashboardOrganizations = config.currentDashboard.data_sources
 
         if w.metadata? && w.metadata.organization_ids?
```

Comment:
Might be nice to flatten this a bit by going:
```coffeescript
return unless w.metadata? && w.metadata.organization_ids?
```

Also I've noticed at the bottom of this callback, `setting.isInitialized = true` is nested within the `if` block, so if there are no `w.metadata.organization_ids`, it will never set itself as `isInitialized`. 

Not sure if this is intended or not. But I think maybe we could move this into a `.finally()` callback?

```coffeescript
ImpacDashboardSvc.load().then(
    (config)->
         ...
).finally(->
   settings.isInitialized = true
)
```

What do you think?

Reply:
@xaun Agree with setting `isInitialized` in `finally` block. Though I'm not yet sure what it's purpose is. I searched for it but it is not used in any views.

Reply:
@MAhsenArif good point, it seems only the widgets-settings/accounts-list setting uses it in its `settings.toMetadata` method. 

I think this was an old framework specific convention of sorts.. @cesar-tonnoir can you confirm if there is any reason to keep this property? It looks like our newer widgets-settings components are not implementing it? If not, we should remove these from the old widgets.